source/_integrations/vallox.markdown

```diff
@@ -51,9 +54,11 @@ automation:
       platform: state
       entity_id: input_select.ventilation_profile
     action:
-      service: vallox.set_profile
+      service: fan.set_preset_mode
```

Comment:
This example isn't needed anymore now?
As in, presets are part of fans, no need to have a input_select for that?

Reply:
I was pondering about this too. In the end, the input select method is one click  shorter than going by way of clicking on the fan entity first, and then on the dropdown in the pop up.

So I left it in there. I thought it doesn’t hurt to keep it, but I’m more or less indifferent.

Reply:
I think we should remove it, as this is not the place to teach people about templates or input selects.

If anything, it will most likely confuse a less experienced use by thinking this is the way to do it.

Reply:
Done 👍